src/style.scss

```diff
@@ -53,3 +60,7 @@ a:hover {
 .detailBorder {
   border-bottom: 1px solid red;
 }
+
+.lineLimit5 {
+  @include lineLimit(5);
+}
```

Comment:
I feel the mixin way is not really useful. Maybe use "Rule of Three" ? http://www.ruanyifeng.com/blog/2013/01/abstraction_principles.html

Reply:
Sure, I just think it might be needed for future.

Reply:
Donald Knuth
> premature optimization is the root of all evil
😈😈😈😈😈

Reply:
Hahaha 😱